### PR TITLE
Prevent actions that affect roles and privileges during rolling upgrade

### PR DESCRIPTION
**Pull request by matriv**
DROP TABLE/VIEW, RENAME TABLE (includes views), SWAP TABLE all touch user/roles privileges, so they must be rejected during a rolling upgrade from < `5.6.0` to >= `5.6.0`, since there is a
UsersMetadata/UserPrivilegesMedata -> RolesMedatada involved, and the nodes < `5.6.0`, cannot read `RolesMetadata`.

### CODE REVIEW

File: docs/appendices/release-notes/5.6.2.rst [hunk 1]
```diff
@@ -45,6 +45,14 @@ See the :ref:`version_5.6.0` release notes for a full list of changes in the
 Fixes
 =====
 
+- Fixed an issue that can cause errors during rolling upgrade from versions <
+  :ref:`version_5.6.0` to >= ``5.6.0`` if one of the following commands:
+  :ref:`DROP TABLE <drop-table>`, :ref:`DROP VIEW <sql-drop-view>`,
+  :ref:`RENAME TABLE <sql-alter-table-rename-to>`, or
+  :ref:`SWAP TABLE <alter_cluster_swap_table>` is executed during the rolling
+  upgraded and before the all nodes are upgraded to :ref:`version_5.6.1`, by
+  rejecting those actions until all nodes are upgraded to >= ``5.6.0``.
```
Comment (seut) [inline docs/appendices/release-notes/5.6.2.rst:54]:
```suggestion
- Fixed an issue that can cause errors during a rolling upgrade to >= :ref:`version_5.6.0`
  if one of the following command is executed on relations with existing privileges:
   - :ref:`DROP TABLE <drop-table>`
   - :ref:`DROP VIEW <sql-drop-view>`
   - :ref:`RENAME TABLE <sql-alter-table-rename-to>`
   - :ref:`SWAP TABLE <alter_cluster_swap_table>`
   These commands are now rejected until all nodes are upgraded.
```

**Review by seut — COMMENTED** [review]
Wouldn't it be nicer to do these checks were they are related to, so inside the `RoleManagerDDLModifier`? And add a comment why we do that?

**Comment by seut** [comment]
We could even be smarter inside `RoleManagerDDLModifier` and throw the exception only when needed -> when privileges are affected...

**Comment by matriv** [comment]
> We could even be smarter inside `RoleManagerDDLModifier` and throw the exception only when needed -> when privileges are affected...

Of course, just followed the easy path :( sorry, I will move them and check conditionally only if there is a modification involved.

File: server/src/main/java/io/crate/execution/ddl/views/TransportDropViewAction.java [hunk 2]
```diff
@@ -38,8 +38,9 @@
 import org.elasticsearch.threadpool.ThreadPool;
 import org.elasticsearch.transport.TransportService;
 
-import java.io.IOException;
-import java.util.List;
+import io.crate.metadata.RelationName;
+import io.crate.metadata.cluster.DDLClusterStateService;
+import io.crate.metadata.view.ViewsMetadata;
```
Comment (seut) [inline server/src/main/java/io/crate/execution/ddl/views/TransportDropViewAction.java:43]: minor: maybe revert this changes as no real change was made

**Review by seut — APPROVED** [review]
👍 thx

File: server/src/main/java/io/crate/execution/ddl/views/TransportDropViewAction.java (hunk 2 above)
Comment (matriv) [reply server/src/main/java/io/crate/execution/ddl/views/TransportDropViewAction.java:43]: yep, of course, slipped through, thx
